dallinger/recruiters.py

```diff
@@ -145,6 +154,26 @@ def recruit_participants(self, n=1):
             duration_hours=self.config.get('duration')
         )
 
+    def notify_recruited(self, participant, experiment):
+        """Assign a Qualification to the Participant for the experiment ID,
+        and for the configured group_name, if it's been set.
+        """
+        score = '1'  # always '1'; doesn't matter
```

Comment:
I think we could change the current behavior from "exclude workers who have X qualification" to "exclude workers with X qualification with a value less than Y" pretty easily. Incrementing the score rather than simply assigning it might be possible. I don't remember if you can query a specific worker's qualification details via the API, and I can't think of a way to increment the score without that.